src/winter_onboarding_2021/fleet_management_service/db/core.clj

```diff
@@ -4,7 +4,8 @@
             [next.jdbc.result-set :as rs]
             [winter-onboarding-2021.fleet-management-service.config :refer [config]]
             [camel-snake-kebab.core :as csk]
-            [next.jdbc.sql :as sql]))
+            [next.jdbc.sql :as sql]
+            [clj-time.jdbc]))
```

Comment:
Look into time coercion

Reply:
Importing `clj-time.jdbc` allows auto-coercing to-and-from joda datetime and jdbc datetime

Reply:
@shhivam https://github.com/clj-time/clj-time/blob/master/src/clj_time/jdbc.clj